dolib/client.py

```diff
@@ -43,8 +43,11 @@ def __init__(self, token: str = None):
         self._load_managers()
 
         self.headers = {
-            "Authorization": "Bearer {token}".format(token=self._token),
+            "Authorization": f"Bearer {self._token}",
             "Content-Type": "application/json",
+            "User-Agent": f"dolib/{__version__}",
+            #  our test lib(vcrpy) for httpx have bugs with gzip and deflate
+            "Accept-Encoding": "",
```

Comment:
I think tt's worth mentioning a `vcrpy` issue here with a `FIXME` marker so we could remove this workaround later.